File: modulo1/segmento3/apunte2/ejercicio2/ejercicio2.py
name = input("Ingrese su nombre de usuario: ")

password = int(input("Ingrese su contraseña: "))

if name == "juan" and password == 1234:
	print("Hola  Juan")
elif name == "pedro" and password == 5678:
	print("Hola Pedro")
else:
	print("Usted no es Pedro y tampoco es Juan")

File: modulo1/segmento3/apunte2/ejercicio2/ejercicio2.java
import java.util.Scanner;
public class ejercicio2
{
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);

		System.out.print("Ingrese su nombre de usuario: ");
		String usuario = teclado.nextLine();

		System.out.print("Ingrese su contraseña: ");
		String pass = teclado.nextLine();

		if ((usuario.equals("Juan") && (pass.equals("1234"))) || (usuario.equals("Pedro") && (pass.equals("5678")))){
			System.out.println("Bienvenido " + usuario);
		}else{
			System.out.println("Usted no es un usuario registrado");
			}
	}
}
